pkg/fanal/types/artifact.go

```diff
@@ -97,6 +97,9 @@ type Package struct {
 
 	// lines from the lock file where the dependency is written
 	Locations []Location `json:",omitempty"`
+
+	// Files installed by the package
+	SystemInstalledFiles []string `json:",omitempty"`
```

Comment:
nit: This field is located under `Package`, so we can simply call it `InstalledFiles`.
```suggestion
	InstalledFiles []string `json:",omitempty"`
```